react/daily-art-prompt/src/components/ImageLayout.js

```diff
@@ -4,37 +4,58 @@ import Image from './Image.js'
 import CommentLayout from './CommentLayout'
 import * as TYPES from '../store/actions'
 
-export const ImageLayout = ({ onDoubleClick, image, onKeyDown, deleteComment }) =>
-  <div className="column">
-    <Image
-      image={image}
-      onDoubleClick={() => onDoubleClick(image.id)}
-    />
-    <div
-      data-testid="likedDiv" className={image.liked ? "likedText" : "hidden"}>
-      Liked
+export const ImageLayout = ({
+  onDoubleClick,
+  image,
+  onKeyDown,
+  deleteComment,
+  cancelCommentClassNames,
+  editComment,
+  editingCommentClassNames
+}) => {
+
+  const handleSubmit = (e, imageId, commentId) => {
+    if (e.keyCode === 13) {
+      const value = e.target.value
+      editComment(imageId, commentId, value)
+    }
+  }
+  return (
+    <div className="column">
+      <Image
+        image={image}
+        onDoubleClick={() => onDoubleClick(image.id)}
+      />
+      <div
+        data-testid="likedDiv" className={image.liked ? "likedText" : "hidden"}>
+        Liked
     </div>
```

Comment:
Try not using css to hide stuff
```js
{image.liked && (
<div data-testid="likedDiv" className="likedText">
        Liked
    </div>	 
)
}
   
```

Reply:
I did this, but now the image sometimes moves up/down depending when the div is showing or not. Is there a way to fix this?